spec/scheduling-tasks.md

```diff
@@ -148,7 +148,7 @@ A <dfn>scheduler task</dfn> is a [=/task=] with an additional numeric
 The following [=task sources=] are defined as <dfn>scheduler task sources</dfn>,
 and must only be used for [=scheduler tasks=].
 
-: <dfn>The posted task task source</dfn>
+: <dfn>The posted task's task source</dfn>
```

Comment:
Not sure about this. This is defining a specific type of task source, i.e. the task source to use for tasks scheduled with the postTask API, and the possession doesn't really fit here (it's a category, so singular possession doesn't make sense to me). Probably "The" should move out of the definition though, and maybe we could consider renaming since we'll probably share this with yield.

Reply:
Yeah I wasn't sure.  I reverted but moved `the` out, and will let you consider rename later.